frontend/src/components/footer/Footer.tsx

```diff
@@ -1,20 +1,91 @@
 const Footer = () => {
-    return (
-      <footer className="bottom-0 w-full">
-          <div className="flex flex-row justify-evenly text-sm text-zinc-500 p-1">
-              <p>About</p>
-              <p>Terms of Service</p>
-              <p>Contact Us</p>          
-              <p>Careers</p>
-              <p>Advertising</p>
-              <p>Privacy Policy</p>
-              <p>Cookie Policy</p>
-              <p>Help Center</p>
-              <p>© 2024 ShopQuik Corp.</p>  
-          </div>
-      </footer>
-    );
-  };
-  
-  export default Footer;
-  
\ No newline at end of file
+  return (
+    <footer className="bg-inherit">
+        {/* <div className="flex flex-row justify-evenly text-sm text-zinc-500 p-1">
+            <p>About</p>
+            <p>Terms of Service</p>
+            <p>Contact Us</p>          
+            <p>Careers</p>
+            <p>Advertising</p>
+            <p>Privacy Policy</p>
+            <p>Cookie Policy</p>
+            <p>Help Center</p>
+            <p>© 2024 ShopQuik Corp.</p>  
+        </div> */}
+      {/* <div className="p-">
+          <a href="">About</a>
+          <a href="">Terms of Service</a>
+          <a href="">Contact Us</a>
+          <a href="">Careers</a>
+          <a href="">Support</a>
+          <a href="">Privacy Policy</a>
+      </div> */}
+  <div className="max-w-full mx-auto mx-5">
```

Comment:
mx-auto and mx-5 contradict each other

```suggestion
  <div className="max-w-full mx-5">
```